snekcord/states/messagestate.py

```diff
@@ -69,11 +69,29 @@ async def create(
             json['tts'] = bool(tts)
 
         if embed is not None:
+            if 'embeds' not in json:
+                json['embeds'] = []
```

Comment:
Is this worth checking? At this point this condition will never be false

Reply:
`json['embeds']` will only exist if `embed` is provided :thinking:

Reply:
but in the `embed` if statement, it doesn't have that key